src/rules.jl

```diff
@@ -195,14 +195,6 @@ struct WirtingerRule{P<:AbstractRule,C<:AbstractRule} <: AbstractRule
     conjugate::C
 end
 
-function WirtingerRule(𝒟::Type, primal::AbstractRule, conjugate::AbstractRule)
```

Comment:
I think we should still keep this, because it is helpful for defining more complex rules, where `@scalar_rule` is not enough. An example would be functions, which take `Vararg`s like `hypot`